src/compiler/worker/runWebpackCompiler.js

```diff
@@ -67,6 +67,8 @@ module.exports = function runWebpackCompiler({
     ],
   });
 
+  // As of Webpack 4.12.0, |outputFileSystem| must be set since there is no
+  // fallback despite the documentation stating otherwise.
```

Comment:
Haha Webpack docs always being helpful on the topic 😄

Reply:
Yeah, my first attempt was removing this line. I was surprised when it simply crashed 😄 

Thanks for reviewing and merging it 🎉